.github/workflows/tests.yml

```diff
@@ -139,6 +139,74 @@ jobs:
           fi
           popd
 
+  test_pywavelets_linux_free_threaded:
+    name: linux-cp${{ matrix.python-version }}t-${{ matrix.OPTIONS_NAME }}
```

Comment:
Let's make this simpler and name it directly: `linux-cp313t-with-scipy`.

Note that there's a second job with `cp310t` in the name that shouldn't be there in the job list right now.